poppy/tests/test_core.py

```diff
@@ -746,7 +746,7 @@ def test_CompoundOpticalSystem():
 
 # Tests for the inwave argument
 
-def test_inwave_fraunhoffer(plot=False):
+def test_inwave_fraunhofer(plot=False):
```

Comment:
There were two misspellings of this, so I'm reasonably confident that they're misspellings.